src/init.cpp

```diff
@@ -798,6 +798,10 @@ bool AppInit2(Config &config, boost::thread_group &threadGroup, CScheduler &sche
     if (fStandard != Params().RequireStandard())
         return InitError(
             strprintf("acceptnonstdtxn is not currently supported for %s chain", chainparams.NetworkIDString()));
+
+    // Set Dust Threshold for outputs.
+    nDustThreshold = GetArg("-dust-threshold", DEFAULT_DUST_THRESHOLD);
```

Comment:
This would be better as a tweak since it should be dynamically adjustable.  Also, no other parameter uses word-word.  like "bytespersigop" we should call this dustthreshold.  I'm concerned that the - may screw up other command line/conf file parsers, even if it works for ours.

Reply:
ok will do